http_check/tests/test_http_check.py

```diff
@@ -3,31 +3,200 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+import os
 
-# 3p
+import pytest
 import mock
 
 from datadog_checks.http_check import HTTPCheck
+from datadog_checks.http_check.http_check import DEFAULT_EXPECTED_CODE
 from datadog_checks.utils.headers import headers as agent_headers
 from .common import (
-    FAKE_CERT, CONFIG, CONFIG_HTTP_HEADERS, CONFIG_SSL_ONLY, CONFIG_EXPIRED_SSL, CONFIG_CUSTOM_NAME, CONFIG_DATA_METHOD,
-    CONFIG_HTTP_REDIRECTS, CONFIG_UNORMALIZED_INSTANCE_NAME, CONFIG_DONT_CHECK_EXP
+    HERE, FAKE_CERT, CONFIG, CONFIG_SSL_ONLY, CONFIG_EXPIRED_SSL, CONFIG_CUSTOM_NAME,
+    CONFIG_DATA_METHOD, CONFIG_HTTP_REDIRECTS, CONFIG_UNORMALIZED_INSTANCE_NAME,
+    CONFIG_DONT_CHECK_EXP
 )
 
 
-def test_http_headers(http_check):
+@pytest.mark.unit
+def test__load_conf(http_check):
     """
-    Headers format.
+    Test the defaults and the pieces of _load_conf that actually perform some logic
     """
-
-    # Get just the headers from http_check._load_conf(...), which happens to be at index 11
-    headers = http_check._load_conf(CONFIG_HTTP_HEADERS['instances'][0])[11]
-
+    # misconfiguration
+    with pytest.raises(Exception) as e:
+        http_check._load_conf({})
+        assert str(e).contains('Bad configuration')
```

Comment:
there isn't a `contains` method, can we use `str(e) not in 'Bad configuration'`?
Can we also explicitly only catch "AssertionError"s here?